src/main/kotlin/minesweeper/domain/Land.kt

```diff
@@ -0,0 +1,21 @@
+package minesweeper.domain
+
+class Land(
+    mines: Mines,
+    val point: Point,
+    private var isOpened: Boolean = false,
+) {
+    val aroundMineCount: Int
+
+    init {
+        aroundMineCount = countAroundMines(mines)
+    }
```

Comment:
init 블럭 없이 한 번에 초기화해도 괜찮을 것 같아요!

```suggestion
    val aroundMineCount: Int = countAroundMines(mines)
```